worlds/dark_souls_3/__init__.py

```diff
@@ -0,0 +1,268 @@
+# world/dark_souls_3/__init__.py
+import json
+import os
+
+from .Options import dark_souls_options  # the options we defined earlier
+from .data.items_data import weapons_upgrade_5_table, weapons_upgrade_10_table, item_dictionary_table, key_items_list
+from .data.locations_data import location_dictionary_table, cemetery_of_ash_table, fire_link_shrine_table, \
+    high_wall_of_lothric, \
+    undead_settlement_table, road_of_sacrifice_table, consumed_king_garden_table, cathedral_of_the_deep_table, \
+    farron_keep_table, catacombs_of_carthus_table, smouldering_lake_table, irithyll_of_the_boreal_valley_table, \
+    irithyll_dungeon_table, profaned_capital_table, anor_londo_table, lothric_castle_table, grand_archives_table, \
+    untended_graves_table, archdragon_peak_table, firelink_shrine_bell_tower_table
+from ..AutoWorld import World, WebWorld
+from BaseClasses import MultiWorld, Location, Region, Item, RegionType, Entrance, Tutorial, ItemClassification
+from ..generic.Rules import set_rule
+
+
+class DarkSouls3Web(WebWorld):
+    tutorials = [Tutorial(
+        "Multiworld Setup Tutorial",
+        "A guide to setting up the Archipelago Dark Souls 3 randomizer on your computer.",
+        "English",
+        "setup_en.md",
+        "setup/en",
+        ["Marech"]
+    )]
+
+
+class DarkSouls3World(World):
+    """
+    Dark souls 3 is an Action role-playing game and is part of the Souls series developed by FromSoftware.
+    Played in a third-person perspective, players have access to various weapons, armour, magic, and consumables that
+    they can use to fight their enemies.
+    """
+
+    game: str = "Dark Souls III"  # name of the game/world
+    options = dark_souls_options  # options the player can set
+    topology_present: bool = True  # show path to required location checks in spoiler
+    remote_items: bool = False  # True if all items come from the server
+    remote_start_inventory: bool = False  # True if start inventory comes from the server
+    web = DarkSouls3Web()
+
+    # data_version is used to signal that items, locations or their names
+    # changed. Set this to 0 during development so other games' clients do not
+    # cache any texts, then increase by 1 for each release that makes changes.
```

Comment:
Those comments can go. They are in the template for explanation, no need to replicate that in worlds.